.github/ISSUE_TEMPLATE/newsletter_box.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Newsletter
```

Comment:
Let's call it 'Newsletter collection box'?

Reply:
sure